### ISSUE TITLE
Missing Value appearing in my alerts in slack.

### ISSUE THREAD

**Issue by GeneralSpacey** [Yelp/elastalert#2068]
I am using Filebeat to read a log file, and then send it to elasticsearch.

My elastalert message is this

`alert_text_type: alert_text_only`
`alert_text: "Triggered at _{}_\n\n*Attacker:*\nIP: {} \nUser-Agent: {}\nDetection: `{}`\n\n:Id: {}\nUser: {}"`
`alert_text_args:`
  `- endpoint.localtime`
  `- context.ip`
  `- context.http.userAgent`
  `- detections_parsed.type`
 ` - _id`
`  - meta.user`

but in the actual slack message it has `<MISSING VALUE>` for all the fields except for _id. How do I add the rest of the fields? I've already done the index creation process.

**Comment by remotephone** [comment]
You need to specify the fields you want mapped in order.

alert_text_type: alert_text_only
alert_text: "Triggered at _{0}_\n\n*Attacker:*\nIP: {1} \nUser-Agent: {2}\nDetection: {3}\n\n:Id: {4}\nUser: {5}"
alert_text_args:
- endpoint.localtime
- context.ip
- context.http.userAgent
- detections_parsed.type
- _id
- meta.user

**Comment by subodhkarwa** [comment]
Yes, you might still get those after updating mapped parameters.
check this thread for solution:
https://github.com/Yelp/elastalert/issues/1260
